autoPyTorch/api/base_task.py

```diff
@@ -1259,7 +1274,8 @@ def _init_ensemble_builder(
             EnsembleBuilderManager
 
         """
-        assert self._logger is not None, "logger should be initialised to fit ensemble"
+        if self._logger is None:
+            raise ValueError("logger should be initialised to fit ensemble")
```

Comment:
It does not matter so much, but I think you often use American English, right?
Then let's go by `initialized`